src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncContext.Unix.cs

```diff
@@ -1082,6 +1082,10 @@ public void SetNonBlocking()
             }
         }
 
+        public bool GetNonBlocking() {
```

Comment:
nit: bracing style doesn't match other methods in file.  should use Allman-style bracing.